gtest/src/program.rs

```diff
@@ -20,6 +20,27 @@ use std::{
 };
 use wasm_instrument::gas_metering::ConstantCostRules;
 
+#[derive(
+    Default,
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    PartialOrd,
+    Ord,
+    derive_more::Add,
+    derive_more::AddAssign,
+    derive_more::Display,
+)]
+pub struct Gas(pub(crate) u64);
```

Comment:
For me, I would also manually derive debug and display to don't show the inner value

Reply:
I can't agree with that, sorry. I think it is helpful to see the gas consumption changes when debugging a program.